helix-term/src/args.rs

```diff
@@ -17,6 +17,8 @@ pub struct Args {
     pub log_file: Option<PathBuf>,
     pub config_file: Option<PathBuf>,
     pub files: Vec<(PathBuf, Position)>,
+    pub line_num: u32,
+    pub commands: Vec<String>,
```

Comment:
Command parsing and executing should be removed or moved to a separate PR: this is something that should be discussed and designed separately. I'm not sure it should be added in general.